src/data/local/github/tracks/track-7/steps/step-1.md

```diff
@@ -1,9 +1,11 @@
-Once you've finished the list, your next task is to create a **user detail screen**, both **UI** and **Integration**, which will show up when you click on a user in user list screen.
+# Step 1 - User Details Screen
 
-## Step 1 - UI
-This screen should have the following information about the user
-- [ ] name
-- [ ] email
-- [ ] role
+The last (but not least) screen is the User Details screen. This screen will shows up when the user clicks in a list item from the User List Screen. 
 
-**Tip**: It's a common practice to reuse some UI components of Detail screen in Create screen. So think if it's possible, in your app, to create the same UI for both detail and create user screen.
+In this step, you need to do both frontend and integration 😮. 
+
+After done, your screen should:
```

Comment:
Aqui não ficou claro pq estamos reusando elementos do create...

Reply:
@tibawatanabe é que colocamos essa dica no step de criação de usuário, que vem antes desse

> **Note:** Later on, we'll create an User Detail Screen and it's a common practice to reuse some UI components of Detail screen in Create screen. So think if is possible, in your app, to create the same UI for both detail and create user screen.

Reply:
Isso eu vi, o que não ficou claro é pq eu deveria reusar, sendo que no detalhe eu só mostro (ou seja, div's e span's cumprem o papel) os dados. Faz sentido se for a tela de edição